build/charts/yorkie-cluster/values-minikube.yaml

```diff
@@ -0,0 +1,152 @@
+# Minimal configuration for minikube testing
+# This focuses on MongoDB deployment without Istio
+
+yorkie:
+  name: yorkie
+  namespace: yorkie
+  autoscaling:
+    enabled: false
+    replicaCount: 1
+  image:
+    repository: yorkieteam/yorkie
+    pullPolicy: IfNotPresent
+    tag: "0.6.41-rc"
+  args:
+    useDefaultProject: false
+    dbConnectionUri: mongodb://admin:admin123@mongodb-mongos.yorkie.svc.cluster.local:27017/yorkie-meta?authSource=admin
+    dbName: yorkie-meta
+  ports:
+    rpcPort: 8080
+    profilingPort: 8081
+  logLevel: "info"
+  mongoMonitoringEnabled: false
+  mongoMonitoringSlowQueryThreshold: "100ms"
+  resources:
+    requests:
+      cpu: 100m
+      memory: 256Mi
+    limits:
+      cpu: 500m
+      memory: 512Mi
+
+# Disable Istio-related features for minikube
+ingressGateway:
+  enabled: false
+
+ingress:
+  enabled: false
+
+ratelimit:
+  enabled: false
+
+# MongoDB configuration with Percona Operator
+mongodb:
+  enabled: true
+  operator:
+    enabled: true
+    watchNamespace: ""
+
+yorkie-mongodb:
+  fullnameOverride: mongodb
+  finalizers:
+    - percona.com/delete-psmdb-pods-in-order
+
+  crVersion: "1.18.0"
+
+  image:
+    repository: percona/percona-server-mongodb
+    tag: "6.0.19-17"
```

Comment:
`6.0.19-17` does not exists, so may need to use something like `6.0.19-16-multi` or `8.0.16`.